app/assets/javascripts/components/GlobalNav.js

```diff
@@ -121,7 +285,15 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities'], function($
       .parents('[data-dough-nav-clumps]').toggleClass('is-active');
   };
 
-  GlobalNav.prototype._openDesktopSubNav = function(index) {
+  GlobalNav.prototype._openDesktopSubNav = function(index, delay) {
+    var timeoutDelay;
+
+    if (delay) {
+      timeoutDelay = this.delay;
+    } else {
+      timeoutDelay = 0;
+    }
```

Comment:
In fact that doesn't quite do the same thing: I've added true and false parameters to determine whether or not we need a delay on the submenu display (i.e. not when it is via a keyboard action). It probably could be less verbose but this breaks some intended functionality.